bin/deferred_work_manager

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env ruby
+$LOAD_PATH << File.dirname(__FILE__) + "/../lib" if $0 == __FILE__
```

Comment:
Also, where is this from? If it's being executed as part of a gem context (which it should always be) then the `lib/` directory will always be on the load path.